_release_notes/armoryspinnaker_v2.18.1.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: post
+title: v2.18.1 Armory Release (OSS Release 1.18.7)
+order: -21820200403172919
+hidden: false
+---
+
+# 04/03/20 Release Notes
+{:.no_toc}
+
+> Note: If you're experiencing production issues after upgrading Spinnaker, rollback to a [previous working version](http://docs.armory.io/admin-guides/troubleshooting/#i-upgraded-spinnaker-and-it-is-no-longer-responding-how-do-i-rollback) and please report issues to [http://go.armory.io/support](http://go.armory.io/support).
+
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+
+## Known Issues
+There are currently no known issues with this release.
+
```

Comment:
Not sure if there is anything to put here?

Reply:
Does this release fix the PE known issue? https://s.armory.io/KouWPo9q

If it's fixed, we should add that it's fixed to the Highlights section 
If not, we should add that.

Reply:
Oh, it does fix the PE issue. So nothing to put here.